maestrowf/maestro.py

```diff
@@ -29,23 +29,25 @@
 
 """A script for launching a YAML study specification."""
 from argparse import ArgumentParser, ArgumentError, RawTextHelpFormatter
-import jsonschema
+
 import logging
 import os
 import shutil
-import six
 import sys
-import tabulate
 import time
 
+import tabulate
```

Comment:
Did your IDE reorder these? I'm noticing a good number of these.

Reply:
fixed